### PR TITLE
[#365] Basic edit

### PR DESCRIPTION
**Pull request by uroboros**
A first line-edit of the Shelley Spec (Formal Spec of the Cardano Ledger) - focussing on typos and grammar

### CODE REVIEW

File: latex/utxo.tex [hunk 1]
```diff
@@ -232,21 +231,21 @@ \subsection{UTxO Transitions}
 
 The accounting for the reward withdrawals is not done in this transition system.
 The rewards are tracked with the delegation state and will
-be removed in the final delegation transition, see ~\ref{eq:delegs-base}.
+be removed in the final delegation transition, see Figure~\ref{eq:delegs-base}.
```
Comment (mgudemann) [inline latex/utxo.tex]: this refers to an equation, not a figure

File: latex/chain.tex [hunk 2]
```diff
@@ -423,8 +423,8 @@ \subsection{Update Nonces Transition}
 
 The transition rule $\mathsf{UPDN}$ takes the slot \var{s} as signal.
 There are two different cases for $\mathsf{UPDN}$, one where \var{s} is not yet
-\SlotsPrior slots from the begin of the epoch, and one where \var{s} is at least
-\SlotsPrior slots after the first slot of the epoch.
+\SlotsPrior  slots from the begin of the epoch and one where \var{s} is at least
+\SlotsPrior  slots after the first slot of the epoch.
```
Comment (mgudemann) [inline latex/chain.tex]: I think having `\SlotsPrior{}` would be better that 2 spaces

File: latex/utxo.tex (hunk 1 above)
Comment (uroboros) [reply latex/utxo.tex]: oops! thanks

File: latex/chain.tex [hunk 3]
```diff
@@ -295,15 +295,15 @@ \subsection{New Epoch Transition}
   map.
 \item The epoch state is updated first with the rewards update \var{ru} and then
   via the call to $\mathsf{EPOCH}$.
-\item The rewards update to set to \Nothing.
+\item The rewards update is set to \Nothing.
```
Comment (uroboros) [inline latex/chain.tex]: not sure about this one

File: latex/chain.tex [hunk 4]
```diff
@@ -528,7 +528,7 @@ \subsection{Operational Certificate Transition}
   \var{vk_{cold}}.
 \end{itemize}
 
-After this, the transition systems updates the operational certificate state by
+After this, the transition system updates the operational certificate state by
```
Comment (uroboros) [inline latex/chain.tex]: perhaps debatable?

File: latex/crypto-primitives.tex [hunk 5]
```diff
@@ -120,7 +120,7 @@ \section{Cryptographic primitives}
   \emph{Constraints}
   \begin{align*}
     & \forall n\in\N, (sk_n, vk) \in \KeyPairEv, ~ d \in \Data,~ \sigma \in \Sig \cdot \\
-    & ~~~~~~~~\signEv{sk}{n}{d} = \sigma \implies \verifyEv{vk}{m}{d}{\sigma}
+    & ~~~~~~~~\signEv{sk}{n}{d} = \sigma \implies \verifyEv{vk}{n}{d}{\sigma}
```
Comment (uroboros) [inline latex/crypto-primitives.tex]: please confirm

File: latex/epoch.tex [hunk 6]
```diff
@@ -668,13 +668,13 @@ \subsection{Protocol Parameter Update Transition}
...
-    deposit pot to cover all possible refunds, and the difference is \textbf{less} than
+    deposit pot to cover all possible refunds and the difference is \textbf{less} than
     the reserve pot, then Rule~\ref{eq:new-pc-accepted} meets the precondition.  Funds are moved
     from the reserve pot to cover the difference and the protocol parameters are updated.
 
   \item If the new protocol parameters mean that \textbf{more} funds are required in the
-    deposit pot to cover all possible refunds, and the difference is \textbf{more} than
-    the reserve pot, then Rule~\ref{eq:new-pc-denied} meets the precondition and no state changes.
+    deposit pot to cover all possible refunds and the difference is \textbf{more} than
+    the reserve pot, then Rule~\ref{eq:new-pc-denied} meets the precondition and there is no change of state.
```
Comment (uroboros) [inline latex/epoch.tex]: or "and there are no state changes" ?

File: latex/epoch.tex [hunk 7]
```diff
@@ -1052,13 +1052,13 @@ \subsection{Rewards Distribution Calculation}
 \clearpage
 
 Figure~\ref{fig:functions:reward-splitting} gives the calculation for
-splitting the pool rewards with its members, as described 6.5.2 of \cite{delegation_design}.
-The portion of rewards allocated to the pool operator and owners is different
+splitting the pool rewards with its members, as described in 6.5.2 of \cite{delegation_design}.
+The portion of rewards allocated to the pool operator and owner is different
```
Comment (uroboros) [inline latex/epoch.tex]: please confirm

File: latex/non-integral.tex [hunk 8]
```diff
@@ -2,7 +2,7 @@ \section{Non-Integral Calculations}
 \label{sec:non-integr-calc}
 
 In the ledger there are several cases where non-integral calculations are
-required. This does concern the delegation transitions, not value transactions.
+required. This concerns the delegation transitions, but not the value transactions.
```
Comment (uroboros) [inline latex/non-integral.tex]: not sure about this - and "value transactions" ?

File: latex/utxo.tex [hunk 9]
```diff
@@ -330,7 +329,7 @@ \subsection{Deposits and Refunds}
 
 Deposits are described in appendix B.2 of the delegation design document \cite{delegation_design}.
 These deposit functions were used above in the UTxO transition,~\ref{sec:utxo-trans}.
-Deposits are used for stake key registration certificate and pool
+Deposits are used for stake key registration and pool
```
Comment (uroboros) [inline latex/utxo.tex]: debatable

File: latex/chain.tex (hunk 3 above)
Comment (JaredCorduan) [reply latex/chain.tex]: good catch, `is` is correct, thanks!

File: latex/chain.tex [hunk 10]
```diff
@@ -423,8 +423,8 @@ \subsection{Update Nonces Transition}
 
 The transition rule $\mathsf{UPDN}$ takes the slot \var{s} as signal.
 There are two different cases for $\mathsf{UPDN}$, one where \var{s} is not yet
-\SlotsPrior slots from the begin of the epoch, and one where \var{s} is at least
-\SlotsPrior slots after the first slot of the epoch.
+\SlotsPrior{} slots from the begin of the epoch and one where \var{s} is at least
```
Comment (JaredCorduan) [inline latex/chain.tex]: "begin" should "beginning" as well. thanks for catching that missing space!

File: latex/chain.tex (hunk 4 above)
Comment (JaredCorduan) [reply latex/chain.tex]: I like the singular better as well.  the "transition system" is a collection of "rules" with a given name and type signature. here there is just one "system" doing the updating.

File: latex/chain.tex [hunk 11]
```diff
@@ -1076,7 +1076,7 @@ \subsection{Chain Transition}
 $\mathsf{UPDN}$, $\mathsf{OCERT}$ and $\mathsf{NEWEPOCH}$ as sub-rules.
 
 The environment for the chain rule is shown in Figure~\ref{fig:ts-types:chain},
-it comprises the current slot number \var{s_{now}} and the new protocol
+it comprises of the current slot number \var{s_{now}} and the new protocol
```
Comment (JaredCorduan) [inline latex/chain.tex]: maybe "it is comprised of"?

File: latex/crypto-primitives.tex (hunk 5 above)
Comment (JaredCorduan) [reply latex/crypto-primitives.tex]: yes, your change is correct, thanks!

File: latex/epoch.tex (hunk 6 above)
Comment (JaredCorduan) [reply latex/epoch.tex]: both are fine with me :)

File: latex/epoch.tex [hunk 12]
```diff
@@ -1278,7 +1278,7 @@ \subsection{Reward Update Calculation}
 \begin{itemize}
   \item The change to the treasury. This will be a positive value.
   \item The change to the reserves. This will be a negative value.
-  \item The net reward pot.
+  \item The new reward pot.
```
[the remark this answers is not in the document]
Comment (JaredCorduan) [reply latex/epoch.tex]: `new` reward pot is correct, thanks!

File: latex/non-integral.tex (hunk 8 above)
Comment (JaredCorduan) [reply latex/non-integral.tex]: hmmm, this sentence is a bit odd (perhaps just outdated). There are no "delegations transactions" or "value transactions" per say. It just that the "delegation components/parts" of a transaction are the culprits for the non-integral calculations, and not the UTxO components.

File: latex/utxo.tex (hunk 9 above)
Comment (JaredCorduan) [reply latex/utxo.tex]: I like the change

File: latex/non-integral.tex [hunk 13]
```diff
@@ -2,7 +2,7 @@ \section{Non-Integral Calculations}
 \label{sec:non-integr-calc}
 
 In the ledger there are several cases where non-integral calculations are
-required. This does concern the delegation transitions, not value transactions.
+required, particularly calculations relating to delegation transitions.
```
Comment (uroboros) [inline latex/non-integral.tex]: @JaredCorduan what do you think of this?

**Comment by uroboros** [comment]
> Thank you for adding some polish to this document @uroboros ! I made several comments below, but it's mostly just me agreeing with you. :) The only thing that I'd do differently (which I mentioned in one of the comments) is to keep the "owners".

thanks @JaredCorduan I fixed "owners" and there was one more tweak, otherwise I'll leave it at this for this PR

File: latex/non-integral.tex (hunk 13 above)
Comment (JaredCorduan) [reply latex/non-integral.tex]: perfect!

**Comment by uroboros** [comment]
> nitpick: [ea8f9c6](https://github.com/input-output-hk/cardano-ledger-specs/commit/ea8f9c6e2a74f9524adc35cda0b0b9b579aa4045) should read `Basic...`

@mgudemann the irony of me spelling "basid" in a commit title focussed on typos is... profound :)

**Comment by mgudemann** [comment]
@uroboros I am sure it was just a test for us :-)

**Pull request merged**